### ISSUE TITLE
Adding Alt text to images

### ISSUE THREAD

**Issue by PaigesterM** [esucsowt/website#69]
This is for accessibility reasons, adding alt text to image tags allows screen readers to read the images to the user. The alt text should be descriptive enough so the user understands what it is and it's context.

**Comment by Moju-juju** [comment]
View pull request #75, all current pictures now have more descriptive alt text
